src/components/player/index.jsx

```
@@ -0,0 +1,180 @@
+      setTotalComments(data?.totalComments || 0);
+
+      setComments(data?.AllComents);
+    } catch (error) {
+      console.error("Error fetching comments:", error);
+    }
+  };
+
+  useEffect(() => {
+    getAllLikes();
+    getAllComments();
+  }, [video?.video_id, dispatch]);
```

ESLint

[error] 'video.video_id' is missing in props validation

(react/prop-types)